SalesforceFoundation/EDA
# [W-9269471] Display latest unmapped record types for auto enrollment program mappings

# Changes

- Pull the latest unmapped record types when creating new auto enrollment program mappings in Education Cloud Settings.

# Issues Closed

-[ W-9269471](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000LnH0YAK/view)

# Testing Notes
Test scenario updated
[T-5147768
](https://gus.lightning.force.com/lightning/r/ADM_Test_Scenario__c/a80AH0000003cIrYAI/view)

Manual Test to be automated in [W-9425585](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000OCgkYAG/view)

1. Select auto enrollment program mappings setting in Education Cloud Settings
2. Click on new button
3. Add a mapping for business organization record type
4. Click Save
5. Click new button
6. Verify the list of available record types - Business Organization record type should not display.
